app/scripts/modules/core/pipeline/config/pipelineConfigurer.html

```diff
@@ -99,12 +99,12 @@
       </div>
       <div class="col-md-9 text-right">
         <span ng-if="viewState.saveError" class="alert alert-danger">
-          There was an error saving your pipeline.
+          <span ng-bind-html="pipelineConfigurerCtrl.getErrorMessage()"></span>
```

Comment:
This, among **many** other reasons, is why I leave the Angular stuff to the professionals.